web/modeshape-web-explorer/pom.xml

```diff
@@ -57,7 +57,12 @@
             <artifactId>gwt-user</artifactId>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
+            <groupId>com.googlecode.gwtmosaic</groupId>
+            <artifactId>gwt-incubator</artifactId>
+            <version>0.4.0-rc2</version>
+            <type>jar</type>
+        </dependency>
```

Comment:
This should be declared in the parent's `<dependencyManagement>` section. `jar` is not required, it's the default type.
Also, isn't there a final version of the dependency we can use ? `rc2` makes me very nervous, not only because it's not final but also because this dependency is not available in Maven Central - which makes it __very exotic__ IMO